components/aiprompt/views/overview.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Overview
+page_title: AIPrompt - Views Overview
+description: Views basics in the AIPrompt for Blazor.
+slug: aiprompt-views-overview
+tags: telerik,blazor,aiprompt,ai,prompt,view,overview
+published: True
+position: 0
+---
+
+# AIPrompt Views
+
+The AIPrompt component provides three different predefined views. There is also the option to define custom views. Through interaction with the component, the user can change the currently active view.
```

Comment:
Are we not worried about the "views" repetition within the first sentence?

Reply:
In the context of technical documentation, repetition is not something that we try to avoid at all times as clarity is a bigger concern. You could rewrite as you wish as long as the text remains unambiguous and easy to read.